source/lib/iows2.js

```diff
@@ -169,7 +169,7 @@ class IOWS2 {
       `Expected first argument productId to be a string, instead ${typeof productId} given. (5492aeea)`
     );
     buCode = String(buCode).trim();
-    productId = String(productId).trim();
+    productId = String(productId).trim().replace(/\./g, '');
```

Comment:
It can happen that the productId contains only dots. As the assertion for the input parameter is above these lines it could happen that a request for an empty productId is made. I’ll change the code so that the assertion checks on the resulting (already trimmed productId

Reply:
Well, nevermind, that’s something that has to fixed in another merge-request.